src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -138,8 +123,8 @@ class WorkspaceSettingsPage extends React.Component {
                                     fallbackIcon={Expensicons.FallbackWorkspaceAvatar}
                                     style={[styles.mb3]}
                                     anchorPosition={{top: 172, right: 18}}
-                                    isUsingDefaultAvatar={!this.state.previewAvatarURL}
-                                    onImageSelected={this.uploadAvatar}
+                                    isUsingDefaultAvatar={!this.props.policy.avatar}
+                                    onImageSelected={file => Policy.updateWorkspaceAvatar(this.props.policy.id, file)}
                                     onImageRemoved={this.removeAvatar}
```

Comment:
Why not also just directly call the Policy action here since we're doing this elsewhere?

```suggestion
                                    onImageRemoved={() => Policy.deleteWorkspaceAvatar(lodashGet(this.props.policy, 'id')}
```